From 1ae39fcc0d66bdcfcdc17087c7f2b264570e2a36 Mon Sep 17 00:00:00 2001
From: "Maciej S. Szmigiero" <mail@maciej.szmigiero.name>
Date: Fri, 31 May 2024 00:05:32 +0200
Subject: [PATCH] hwdb: Add Logitech MX Master 3S Bluetooth ID

We already have an entry for this mouse when it is connected via USB Bolt
receiver, now add a Bluetooth LE connection entry too.

This mouse model supports both connection types, with up to 3 connection
slots (to different hosts) per single mouse.

(cherry picked from commit 7e3cfd15f28dd0093502332d7f3e3fb491b78c7a)

Related: RHEL-5950
---
 hwdb.d/70-mouse.hwdb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 6264268e37..0379eecf07 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -540,8 +540,9 @@ mouse:bluetooth:v046dpb019:name:MX Master 2S Mouse:*
  MOUSE_WHEEL_CLICK_COUNT=24
  MOUSE_WHEEL_CLICK_COUNT_HORIZONTAL=14
 
-# Logitech MX Master 3S (via Bolt Receiver)
+# Logitech MX Master 3S (via Bolt Receiver and Bluetooth)
 mouse:usb:v046dpc548:name:Logitech USB Receiver Mouse:*
+mouse:bluetooth:v046dpb035:name:MX Master 3S B Mouse:*
  MOUSE_DPI=1000@142
 
 # Logitech MX Ergo
